docs/platforms/android/troubleshooting/shared-environments/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Shared Environments
```

Comment:
`m`: title should be more descriptive, also not sure if troubleshooting is the ideal parent page - maybe usage or data management? wdty?

Reply:
:+1: I've moved it to the bottom of our configuration section. Title is now "Using Sentry within an SDK"